Lib/fontParts/objects/base/validators.py

```diff
@@ -115,10 +115,11 @@ def validateLayerName(value):
 # -----
 
 def validateGlyphName(value):
-    """
-    XXX implement
-    """
-    return value
+    if not isinstance(value, basestring):
+        raise FontPartsError("Glyph names must be unicode strings, not %s." % type(value).__name__)
```

Comment:
> This is the thing, the UFO 3 spec doesn't place limits on the glyph name; should fontParts follow that, or should it follow font specs (OT, CFF, Type1, etc which are just ascii)?

We should not limit them to what OTF or whatever allows. I want to get to the point where we can name our glyphs “ instead of quotedblleft.